packages/flutter/lib/src/material/navigation_rail.dart

```diff
@@ -667,9 +668,9 @@ class _RailDestination extends StatelessWidget {
                       heightFactor: 1.0,
                       widthFactor: extendedTransitionAnimation.value,
                       alignment: AlignmentDirectional.centerStart,
-                      child: Opacity(
+                      child: FadeTransition(
                         alwaysIncludeSemantics: true,
-                        opacity: _extendedLabelFadeValue(),
+                        opacity: labelFadeAnimation,
```

Comment:
This is odd: previously, the animation was driven by extendedTransitionAnimation, now it is destinationAnimation. Why this change?

Reply:
copy-paste mistake, fixed